src/program.rs

```diff
@@ -34,22 +34,11 @@ use std::io::{self, Cursor, Read};
 use std::result::Result;
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
-pub enum ColorOpt {
-    Auto,
-    Always,
-    Never,
-}
-
-impl std::str::FromStr for ColorOpt {
-    type Err = &'static str;
+pub struct ColorOpt(pub(crate) clap::ColorChoice);
```

Comment:
I wonder if we should just get rid of `ColorOpt` entirely. Right now it's just wrapping `clap::ColorChoice`, and the inner value is public anyway (so we are tied to `clap::ColorChoice` for the public API already). If we don't implement any trait on it, I don't think there's any advantage in keeping the wrapper.

Reply:
Unfortunately we can't. Like I said [above](https://github.com/tweag/nickel/pull/1309#discussion_r1199591236), we need to implement `From<ColorOpt>` for two other libs' own color types, and we can't implement `From<Clap::ColorChoice>` instead due to orphan rules.

Reply:
Ah, I missed that. Thanks!